bot/controllers/tos.js

```diff
@@ -0,0 +1,31 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+
+class TosController extends BaseController {
+  constructor(message) {
+    // Call BaseController constructor
+    super(message);
+
+    // Aliasing 'this' as controller to allow for binding in actions
+    const controller = this;
+
+    // Array of all commands, see baseCommand.js for prototype
+    this.commands = [
+      new Command(
+        '!tos',
+        '!tos',
+        'Terms of Service',
+        'Send Copy of the Terms of Service',
```

Comment:
Where is this copy sent? Does it send it to the channel? We need to shape our messages to the user carefully and accurately so that we craft an expected user experience.